docs/content/docs/dev/table/sqlClient.md

```diff
@@ -601,6 +601,24 @@ This configuration:
 
 <span class="label label-danger">Attention</span> Compared to the interactive mode, SQL Client will stop execution and exits when there are errors.
 
+### Deploy SQL Files in an Application Cluster
+
+SQL Client also supports deploying a SQL script file to an Application Cluster with the `-f` option, if you specify the deployment target in the config.yaml or startup options.
+Here is an example to deploy script file in an Application Cluster.
+
+```bash 
+./bin/sql-client.sh -f oss://path/to/script.sql \
+      -Dexecution.target=kubernetes-application \
+      -Dkubernetes.cluster-id=${CLUSTER_ID} \
+      -Dkubernetes.container.image.ref=${FLINK_IMAGE_NAME}'
+```
+
+After execution, SQL Client print the cluster id on the terminal. The script can contain any statement that is supported by Flink. But Application cluster only supports one job, please refer to the 
```

Comment:
> any statement that is supported by Flink. But Application cluster only supports one job

Are you sure this limitation is correct? If the script contains multiple DML statements, shouldn’t it be possible to submit multiple jobs in one application cluster ?

Reply:
I just mean the new feature supports to run any kinds of statement, but it doesn't mean the combination of the sql statements. 

If the cluster doesn't enable ha mode, the cluster is able to run multiple jobs.